plugins/inputs/pgbouncer/README.md

```diff
@@ -0,0 +1,61 @@
+# Pgbouncer plugin
+
+This pgbouncer plugin provides metrics for your pgbouncer connection information.
+
+### Configuration:
+
+```toml
+# Description
+[[inputs.pgbouncer]]
+  ## specify address via a url matching:
+  ##   postgres://[pqgotest[:password]]@localhost:port[/dbname]\
+  ##       ?sslmode=[disable|verify-ca|verify-full]
+  ## or a simple string:
+  ##   host=localhost user=pqotest port=... password=... sslmode=... dbname=...
+  ##
+  ## All connection parameters are optional, except for dbname,
+  ## you need to set it always as pgbouncer.
+  address = "host=localhost user=postgres port=6432 sslmode=disable dbname=pgbouncer"
+
+  ## A list of databases to pull metrics about. If not specified, metrics for all
+  ## databases are gathered.
+  # databases = ["app_production", "testing"]
+```
+
+### Measurements & Fields:
+
+Pgbouncer provides two measurement named "pgbouncer_pools" and "pgbouncer_stats", each have the fields as below:
```

Comment:
Make sure to document tags and stick closely to the format in the [EXAMPLE_README.md](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/EXAMPLE_README.md)